File: Try_GameEngine5/src/com/example/try_gameengine/remotecontroller/custom/CustomTouch.java
package com.example.try_gameengine.remotecontroller.custom;

import android.view.MotionEvent;

import com.example.try_gameengine.framework.ALayer;

public class CustomTouch {
	private ALayer touch;
	private MotionEvent event;
	
	public CustomTouch(ALayer touch, MotionEvent event) {
		this.touch = touch;
		this.event = event;
	}
	
	public ALayer getTouch() {
		return touch;
	}
	public void setTouch(ALayer touch) {
		this.touch = touch;
	}
	public MotionEvent getEvent() {
		return event;
	}
	public void setEvent(MotionEvent event) {
		this.event = event;
	}
	
	
}

File: Try_GameEngine5/src/com/example/try_gameengine/remotecontroller/custom/CustomTouch.kt
package com.example.try_gameengine.remotecontroller.custom

import android.view.MotionEvent
import com.example.try_gameengine.framework.ALayer

class CustomTouch(touch: ALayer?, event: MotionEvent?) {
    private var touch: ALayer?
    private var event: MotionEvent?

    init {
        this.touch = touch
        this.event = event
    }

    fun getTouch(): ALayer? {
        return touch
    }

    fun setTouch(touch: ALayer?) {
        this.touch = touch
    }

    fun getEvent(): MotionEvent? {
        return event
    }

    fun setEvent(event: MotionEvent?) {
        this.event = event
    }
}
